broker/data-access-layer/quota/test/quota.QuotaManager.spec.js

```diff
@@ -243,7 +243,7 @@ describe('quota', () => {
           },
           'tags': [],
           'service_guid': 'fe503379-832b-4d56-ad12-3a6ef68dcb34',
-          'space_url': '/v2/spaces/6b48f3ea-0ef1-44eb-9de4-942d89779d37',
+          'space_url': '/v3/spaces/6b48f3ea-0ef1-44eb-9de4-942d89779d37',
```

Comment:
```suggestion
          'space_url': '/v2/spaces/6b48f3ea-0ef1-44eb-9de4-942d89779d37',
```
This is a response of a v2 call. So the link here will be v2.